### PR TITLE
add missing polyfills and modules

### PR DESCRIPTION
**Pull request by benschac**
## What does this PR do and why?

Adds the missing polyfills that were removed in the project via the webpack@5 upgrade via rewired. The additional context into this PR can be found in depending PRs linked by graphites auto-generated comment below.

## Screenshots or screen recordings

N/A but if you run a before / after you'll see significantly fewer errors in the console.

## Acceptance checklist

- [x] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [x] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).

### CODE REVIEW

**Comment by benschac** [comment]
Current dependencies on/for this PR:
* main
  * **PR #867** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/867" target="_blank">Graphite</a> 
    * **PR #868** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/868" target="_blank">Graphite</a> 
      * **PR #869** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/869" target="_blank">Graphite</a>  (this PR)
        * **PR #870** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/870" target="_blank">Graphite</a> 
          * **PR #871** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/871" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/869?utm_source=stack-comment).

File: config-overrides.js
Comment (benschac) [inline config-overrides.js:5]: We don't have any postcss plugins or anything configured, so I figured leaving this blank was the right option.

File: config-overrides.js
Comment (benschac) [inline config-overrides.js:8]: I went through each error and added the polyfill the package error message requested/said it needed/depended on.

**Review by johnnyd-eth — APPROVED** [review]
Legend for picking these up @benschac, not sexy but extremely essential work. LGTM but will wait for a final signoff from @aeolianeth on Monday before merging

File: config-overrides.js
Comment (aeolianeth) [reply config-overrides.js:8]: @benschac amaaaazing. This part is super informative. Speaks to the power of rewire.

**Pull request merged**
